mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -316,6 +318,9 @@
 
     private Optional<SshPublicKey> sshPublicKey;
 
+    @Value("${auth.mock.azure.disk.SSEWithCMk.enable}")
```

Comment:
please add this property to the application.yml

Reply:
Thank you for correction, Updated application.yml - keeping in mind for future updates for other resources other than disk.